app/controllers/api/account.php

```diff
@@ -1500,17 +1500,18 @@
         $response->dynamic($user, Response::MODEL_USER);
     });
 
-App::delete('/v1/account')
-    ->desc('Delete Account')
+App::patch('/v1/account')
```

Comment:
```suggestion
App::patch('/v1/account/status')
```

Completely missed this one. This is the convention we use to update other attributes as well, we may also want to reorder the routes so all patch methods are grouped together.